flake.nix

```diff
@@ -33,7 +33,7 @@
   };
 
   inputs.libparse.inputs.nixpkgs.follows = "nix-eda/nixpkgs";
-  inputs.ioplace-parser.inputs.nixpkgs.follows = "nix-eda/nixpkgs";
+  inputs.ioplace-parser.inputs.nix-eda.follows = "nix-eda";
```

Comment:
what is the impact of this change?

Reply:
the updated ioplace-parser uses a version of the antlr4-runtime fixed explicitly in nix-eda. this allows two projects using antlr4-runtime to be in the same environment, which is necessary to include things like nl2bench.